pages/common/bower.md

```diff
@@ -0,0 +1,32 @@
+# bower
+
+> A package manager optimized for the front-end web development.
+> A package can be a GitHub shorthand, a Git endpoint, a URL, a registered package.
```

Comment:
suggestion: "or a registered package". Also, "github shorthand" may not be entirely clear at first. I wonder if we can come up with a more self-evident, yet still concise, expression to mean the same. Any ideas?